### PR TITLE
Changed install docs from sites framework to REQUEST_BASE_URL

### PR DESCRIPTION
**Pull request by mikaraunio**

### CODE REVIEW

File: docs/index.txt [hunk 1]
```diff
@@ -26,7 +26,7 @@ Once you have installed ``django-request`` you can add it to a django project by
 
     #. Add ``request.middleware.RequestMiddleware`` to ``MIDDLEWARE``. If you use ``django.contrib.auth.middleware.AuthenticationMiddleware``, place ``RequestMiddleware`` after it. If you use ``django.contrib.flatpages.middleware.FlatpageFallbackMiddleware`` place ``request.middleware.RequestMiddleware`` before it else flatpages will be marked as error pages in the admin panel.
 
-    #. Make sure that the domain name in django.contrib.sites admin is correct. This is used to calculate unique visitors and top referrers.
+    #. Add ``REQUEST_BASE_URL`` to your settings with the base URL of your site (e.g. ``https://www.my.site/``). This is used to calculate unique visitors and top referrers. ``REQUEST_BASE_URL`` defaults to ``'http://%s' % Site.objects.get_current().domain``.
```
Comment (felixxm) [inline docs/index.txt:29]: Can you wrap at 79 chars? I try to wrap new docs.

File: README.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ Installation
 - Put `'request'` in your `INSTALLED_APPS` setting.
 - Run the command `manage.py migrate`.
 - Add `request.middleware.RequestMiddleware` to `MIDDLEWARE`. If you use `django.contrib.auth.middleware.AuthenticationMiddleware`, place the `RequestMiddleware` after it. If you use `django.contrib.flatpages.middleware.FlatpageFallbackMiddleware` place `request.middleware.RequestMiddleware` before it else flatpages will be marked as error pages in the admin panel.
-- Make sure that the domain name in `django.contrib.sites` admin is correct. This is used to calculate unique visitors and top referrers.
+- Add `REQUEST_BASE_URL` to your settings with the base URL of your site (e.g. `https://www.my.site/`). This is used to calculate unique visitors and top referrers. `REQUEST_BASE_URL` defaults to `'http://%s' % Site.objects.get_current().domain`.
```
Comment (felixxm) [inline README.md:42]: Can you wrap at 79 chars? I try to wrap new docs.

**Review by felixxm — COMMENTED** [review]
@mikaraunio Thanks :+1:

File: docs/index.txt (hunk 1 above)
Comment (mikaraunio) [reply docs/index.txt]: Done.

File: README.md (hunk 2 above)
Comment (mikaraunio) [reply README.md]: Done.

**Comment by felixxm** [comment]
@mikaraunio Thanks :+1: Merged in 5399983c12afe3dc265244244a75a089f4a6bf42.

**Pull request closed without merging**